api_db.php

```diff
@@ -382,13 +382,6 @@ function parseDBData($results, $interval, $from, $until) {
 				// $data[timestamp] = value_in_this_interval
 				$data[$row[0]] = intval($row[1]);
 			}
-
-			// Fill the missing intervals with zero
```

Comment:
With bar graphs, we do not need to fill gaps (there will simply be gaps for empty days). This change enhances processing speed notably.